### PR TITLE
Change "Annoteren" to "Classificeren"

### PR DESCRIPTION
**Pull request by Verschoofwb**
For a project in the Netherlands, we are working on a project for school kids. We would like to have the word 'Annotate' be translated into 'Classificeren' instead of 'Annoteren'. The latter is a slightly archaic word hardly used in Dutch, while the former is used much more (and children will recognize it).

Staging branch URL: https://pr-{NUMBER}.pfe-preview.zooniverse.org

Fixes # .

changed the translation of Annotate from "Annoteren" to "Classificeren" (classify)

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

**Comment by marten** [comment]
(Context: I'm a fellow native speaker and ex-Zooniverse staff)

Both words have slightly different meaning but I'm okay with either one. But it's worth noting that a prior commit from a project owner was in favour of changing wording from "Classificeer" into "Annoteren' (as well as adding a whole bunch of missing translations).

Additionally, I have no say over this anymore, but I expect the file rename to be a blocking issue which you should probably undo before this would be merged.

**Comment by Verschoofwb** [comment]
Changed the name back to the original.

File: app/locales/nl.js [hunk 1]
```diff
@@ -167,7 +167,8 @@ export default {
...
+      thirdParties: '**What is shared with third parties**\n\nWe will never release e-mail addresses to third parties without your express permission. We will also never share data we collect about you unless (a) it c
+      t be associated with you or your username, and (b) it is necessary to accomplish our research goals. Specifically, we may share your anonymized data with research study participants, other researchers, or in scholarly work describing our research. For example, we might use one of your classifications as an illustration in a paper, show some of your classifications to another user to see if they agree or disagree, or publish statistics about user interaction.\n\nContributions you make to the Talk pages are widely available to others. Aside from the above, information is held as confidentially as is practical within our secured database.',
```
Comment (camallen) [inline app/locales/nl.js:171]: I think this was just a mistake, i'm going to add a commit to fix it so we can focus on the actual intended change in this PR
```suggestion
      thirdParties: '**What is shared with third parties**\n\nWe will never release e-mail addresses to third parties without your express permission. We will also never share data we collect about you unless (a) it cannot be associated with you or your username, and (b) it is necessary to accomplish our research goals. Specifically, we may share your anonymized data with research study participants, other researchers, or in scholarly work describing our research. For example, we might use one of your classifications as an illustration in a paper, show some of your classifications to another user to see if they agree or disagree, or publish statistics about user interaction.\n\nContributions you make to the Talk pages are widely available to others. Aside from the above, information is held as confidentially as is practical within our secured database.',
```

**Comment by camallen** [comment]
thanks @Verschoofwb for the translation 👍

**Pull request merged**